ci/jenkins/test.sh

```diff
@@ -57,18 +56,17 @@ PRINT_DOCKER_STATUS="docker system df -v"
 PRINT_CONTAINERD_STATUS="crictl ps --state Exited"
 
 _usage="Usage: $0 [--kubeconfig <KubeconfigSavePath>] [--workdir <HomePath>]
-                  [--testcase <windows-install-ovs|windows-containerd-conformance|windows-conformance|windows-containerd-networkpolicy|windows-networkpolicy|windows-containerd-e2e|windows-e2e|e2e|conformance|networkpolicy|multicast-e2e>]
+                  [--testcase <windows-install-ovs|windows-conformance|windows-networkpolicy|windows-e2e|e2e|conformance|networkpolicy|multicast-e2e>]
 
 Run K8s e2e community tests (Conformance & Network Policy) or Antrea e2e tests on a remote (Jenkins) Windows or Linux cluster.
 
         --kubeconfig             Path of cluster kubeconfig.
         --workdir                Home path for Go, vSphere information and antrea_logs during cluster setup. Default is $WORKDIR.
         --testcase               Windows install OVS, Conformance and Network Policy or Antrea e2e testcases on a Windows or Linux cluster. It can also be flexible ipam or multicast e2e test.
         --registry               The docker registry to use instead of dockerhub.
-        --proxyall               Enable proxyAll to test AntreaProxy.
```

Comment:
curious as to why we are removing this flag? I know that proxyAll is required on Windows, but isn't this script used for testing proxyAll on Linux as well?

Reply:
yes we missed that part, but proxyall is kind of stale flag in this file, because none of the linux related test imports this file with --proxyall flag, so the part of the code related to proxyall set to true in `deliver_antrea` function is also a stale code, because in all the linux related test only `generate-manifest.sh` and `test-e2e-kind.sh`are called with proxy-all enabled, so i think that we can remove the flag and all the related stale code from `deliver_antrea` function in this file.
@antoninbas @XinShuYang what's your take on this.

Reply:
I will let @XinShuYang make the decision here

Reply:
@XinShuYang , 
Better to separate out removing of the --proxyall from this PR scope @jainpulkit22 can create an another PR for same.

Reply:
> @XinShuYang , Better to separate out removing of the --proxyall from this PR scope @jainpulkit22 can create an another PR for same.

Yes, I believe that `proxyall` is still necessary for the Linux test. In this PR, we can only ensure that it won't affect tests related to Windows.

Reply:
Yes it is not impacting linux.

Reply:
@XinShuYang I see contradictory information in this conversation. Should the flag be preserved or not?

Reply:
@antoninbas
 proxy-all flag is not used on windows, so there is no need to preserve, linux impact is not there wrt these changes in this PR as verified by @jainpulkit22

Reply:
> @XinShuYang I see contradictory information in this conversation. Should the flag be preserved or not?
 @rajnkamr @jainpulkit22 
I missed this change in the last round of review. What I mean is, Windows test does not need the 'proxyall'. But given that test.sh is shared by kind, Linux, and Windows tests, 'proxyall' might still be used in other tests in the future. Therefore, I prefer to preserve this flag in test.sh.

Reply:
@jainpulkit22 @XinShuYang 
Even though windows does not need '--proxy-all' field, We can leave '-proxyall' field intact here.